code/index.html

```diff
@@ -1,22 +1,218 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <!--Google Font Abril Fatface-->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Open+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/e056cb308d.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Örebronx</title>
   </head>
+
   <body>
+    <!-- Header -->
     <header>
-      <!-- Some kind of logo and navbar -->
+      <section class="header-container">
+        <h1>Hitchhiker's guide to Örebronx</h1>
+        <img
+          src="./img/lovetravel.svg"
+          alt="love_travel"
```

Comment:
Try to use the alt as a better way to describe what the image actually shows, here's some reading for future projects:
https://www.w3schools.com/tags/att_img_alt.asp